Client/mods/deathmatch/logic/luadefs/CLuaDrawingDefs.cpp

```diff
@@ -402,6 +403,67 @@ int CLuaDrawingDefs::DxDrawRectangle(lua_State* luaVM)
     return 1;
 }
 
+int CLuaDrawingDefs::DxDrawCircle(lua_State* luaVM)
+{
+    CVector2D vecPosition;
+    float     fRadius;
+    float     fStartAngle;
+    float     fStopAngle;
+    SColor    color;
+    SColor    colorCenter;
+    short     fSegments;
+    float     fRatio;
+    bool      bPostGUI;
+    bool      bSubPixelPositioning;
+
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadVector2D(vecPosition);
+    argStream.ReadNumber(fRadius);
+    argStream.ReadNumber(fStartAngle, 0);
+    argStream.ReadNumber(fStopAngle, 360);
+    argStream.ReadColor(color, 0xFFFFFFFF);
+    argStream.ReadColor(colorCenter, color);
+    argStream.ReadNumber(fSegments, 32);
+    argStream.ReadNumber(fRatio, 1);
+    argStream.ReadBool(bPostGUI, false);
+    argStream.ReadBool(bSubPixelPositioning, false);
```

Comment:
bSubPixelPositioning is for backwards compatibility with old dxDraw functions (which used integer positions). It is not required for new dxDraw functions